src/app/pages/common/pages-not-found-page/pages-not-found-page.component.html

```diff
@@ -0,0 +1,9 @@
+<div class="container not-found">
```

Comment:
Path to this file should be `/pages/not-found`. Also for home it should be `/pages/home`, it should be the same or very similar in some way to routes we will use to easily find specific page.